### PR TITLE
expose prometheus endpoints in ovn-kubernetes to prometheus server

### PR DESCRIPTION
**Pull request by JacobTanenbaum**
along with ovn-kubernetes PR https://github.com/ovn-org/ovn-kubernetes/pull/837

this will allow prometheus to collect metrics from ovn-kubernetes

### CODE REVIEW

**Comment by squeed** [comment]
/hold
until upstream PR is merged.

File: pkg/network/ovn_kubernetes.go [hunk 1]
```diff
@@ -113,6 +113,19 @@ func fillOVNKubernetesDefaults(conf, previous *operv1.NetworkSpec, hostMTU int)
 		}
 		sc.MTU = &mtu
 	}
+
```
Comment (squeed) [inline pkg/network/ovn_kubernetes.go]: This isn't necessary - there's no kube-proxy-config for ovn-kubernetes.

**Comment by squeed** [comment]
@JacobTanenbaum you'll need to rebase and edit the daemonset to pass the right `--metrics-bind-address`.

File: pkg/network/ovn_kubernetes.go (hunk 1 above)
Comment (JacobTanenbaum) [reply pkg/network/ovn_kubernetes.go]: Done

**Comment by squeed** [comment]
much better.
/test e2e-aws-ovn-kubernetes

File: bindata/network/ovn-kubernetes/ovnkube-master.yaml [hunk 2]
```diff
@@ -193,7 +193,8 @@ spec:
             --nodeport \
             --loglevel "${OVN_KUBE_LOG_LEVEL}" \
             ${hybrid_overlay_flags} \
-            --logfile /dev/stdout
```
Comment (squeed) [inline bindata/network/ovn-kubernetes/ovnkube-master.yaml]: Can you please add this port (and the one below) to the pod spec, so we claim the port?

**Comment by squeed** [comment]
duh, ovn-k won't pass until a release is cut
/lgtm

**Pull request merged**